src/harness/certs/revoke_and_generate_crl.sh

```diff
@@ -2,53 +2,79 @@
 
 mkdir -p crl
 
-function blacklist_certificate()
+function revoke_certificate()
 {
-  #Argument1: Name of the certificate to be  blacklisted.
-  #Argument2: Intermediate CA to generate CRL.
+  #Argument1: Name of the certificate to be revoked.
 
+  #Fetch the Common Name to find out the issuer
+  CN=`openssl x509 -issuer -in $1 -noout | sed 's/^.*CN=//'`
```

Comment:
`CN` and `CA` should be [local variables](http://tldp.org/LDP/abs/html/localvar.html).

Reply:
Changed   as local variables